src/util/pelz_io.c

```diff
@@ -280,3 +287,85 @@ int decodeBase64Data(unsigned char *base64_data, size_t base64_data_size, unsign
   BIO_free_all(bio64);
   return (0);
 }
+
+int write_to_pipe(char *msg)
+{
+  int fd;
+  int ret;
+
+  if (file_check((char*) PELZFIFO))
+  {
+    pelz_log(LOG_ERR, "Pipe not found");
+    return 1;
+  }
+
+  fd = open(PELZFIFO, O_WRONLY);
```

Comment:
Should check that the ```open``` call didn't fail.

Reply:
Added check for the open call and the close call.